qemu/tests/blockdev_mirror_vm_reboot.py

```diff
@@ -0,0 +1,29 @@
+from provider.blockdev_mirror_parallel import BlockdevMirrorParallelTest
+
+
+class BlockdevMirrorVMRebootTest(BlockdevMirrorParallelTest):
+    """do block-mirror and vm stop/cont in parallel"""
+
+    def reboot_vm(self):
+        """Reboot VM with qmp command"""
+        self.main_vm.reboot(method="system_reset")
+
+
+def run(test, params, env):
+    """
+    Basic block mirror during vm reboot -- only system disk
+
+    test steps:
+        1. boot VM
+        2. create a file on system disk
+        3. add a target disk for mirror to VM via qmp commands
+        4. do block-mirror for system disk and vm reboot in parallel
+        5. check the mirrored disk is attached
+        6. restart VM with the mirrored disk, check the file and md5sum
```

Comment:
After this step, the case boots up the VM with the original system disk, can we avoid it?

Reply:
Hi Nini, not all tests will perform this step, only tests that write files on the original system disk will perform this step, that's because we write files in /var/tmp, which will never be removed after VM reboot, so the capability of /var/tmp will become smaller and smaller, finally, all space runs out....so we should remove all testing files from the original system disk. After block mirror, the mirror image is in use, so we have to restart VM with the original system disk.

Reply:
Got it. Then LGTM.